CloudSQL: The SQL Server tests are failing

They are consistently failing, both in Windows and Linux (sample [CI output](https://source.cloud.google.com/results/invocations/c0d30f8b-04a5-46eb-af49-4abc1702016f/targets/github%2Fdotnet-docs-samples%2Fcloud-sql%2Fsql-server/tests))

Error here:
```
Kestrel failed to start.
NotSpecified: (:String) [], RemoteException
Unhandled Exception: System.Data.SqlClient.SqlException: A connection was successfully established with the server, but then an error occurred during the pre-login handshake. (provider: TCP Provider, error: 35 - An internal exception was caught) ---> System.IO.IOException: Unable to read data from the transport connection: Connection reset by peer. ---> System.Net.Sockets.SocketException: Connection reset by peer
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   --- End of inner exception stack trace ---
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.Data.SqlClient.SNI.SNITCPHandle.Receive(SNIPacket& packet, Int32 timeoutInMilliseconds)
   --- End of inner exception stack trace ---
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at CloudSql.StartupExtensions.<>c__DisplayClass0_0.<OpenWithRetry>b__0() in /tmpfs/src/github/dotnet-docs-samples/cloud-sql/sql-server/Startup.cs:line 90
   at Polly.Policy.<>c__DisplayClass108_0.<Execute>b__0(Context ctx, CancellationToken ct) in /_/src/Polly/Policy.ExecuteOverloads.cs:line 18
   at Polly.Policy.<>c__DisplayClass138_0.<Implementation>b__0(Context ctx, CancellationToken token) in /_/src/Polly/Policy.SyncNonGenericImplementation.cs:line 18
   at Polly.Retry.RetryEngine.Implementation[TResult](Func`3 action, Context context, CancellationToken cancellationToken, ExceptionPredicates shouldRetryExceptionPredicates, ResultPredicates`1 shouldRetryResultPredicates, Action`4 onRetry, Int32 permittedRetryCount, IEnumerable`1 sleepDurationsEnumerable, Func`4 sleepDurationProvider) in /_/src/Polly/Retry/RetryEngine.cs:line 35
   at Polly.Retry.RetryPolicy.Implementation[TResult](Func`3 action, Context context, CancellationToken cancellationToken) in /_/src/Polly/Retry/RetryPolicy.cs:line 35
   at Polly.Policy.Implementation(Action`2 action, Context context, CancellationToken cancellationToken) in /_/src/Polly/Policy.SyncNonGenericImplementation.cs:line 18
   at Polly.Policy.Execute(Action`2 action, Context context, CancellationToken cancellationToken) in /_/src/Polly/Policy.ExecuteOverloads.cs:line 73
   at CloudSql.StartupExtensions.OpenWithRetry(DbConnection connection) in /tmpfs/src/github/dotnet-docs-samples/cloud-sql/sql-server/Startup.cs:line 82
   at CloudSql.StartupExtensions.InitializeDatabase() in /tmpfs/src/github/dotnet-docs-samples/cloud-sql/sql-server/Startup.cs:line 98
   at CloudSql.Program.Main(String[] args) in /tmpfs/src/github/dotnet-docs-samples/cloud-sql/sql-server/Program.cs:line 37
BUILD FAILED
```

I'm skipping the tests in #1469 until they are fixed.

--- a/cloud-sql/sql-server/runTestsWithSqlServer.ps1
+++ b/cloud-sql/sql-server/runTestsWithSqlServer.ps1
@@ -11,43 +11,40 @@
 # WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 # License for the specific language governing permissions and limitations under
 # the License.
-
-
-# Skipping because of https://github.com/GoogleCloudPlatform/dotnet-docs-samples/issues/1471
-
-#Import-Module -DisableNameChecking ..\..\BuildTools.psm1
-
+ 
+Import-Module -DisableNameChecking ..\..\BuildTools.psm1
+ 
 # Download and run the sql proxy.
-#if (-not (Test-Path cloud_sql_proxy.exe)) {
-#	if ($PSVersionTable.Platform -eq 'Unix') {
-#		Invoke-WebRequest -Uri 'https://dl.google.com/cloudsql/cloud_sql_proxy.linux.amd64' -OutFile cloud_sql_proxy.exe
-#		chmod +x cloud_sql_proxy.exe
-#	} else {
-#		Invoke-WebRequest -Uri 'https://dl.google.com/cloudsql/cloud_sql_proxy_x64.exe' -OutFile cloud_sql_proxy.exe
-#	}
-#}
-#$proxy = Start-Job -ArgumentList (Get-Location) -ScriptBlock {
-#	Set-Location $args[0]
-#	./cloud_sql_proxy.exe --instances=$env:TEST_CLOUDSQL2_SQLSERVER_INSTANCE=tcp:1433 `
-#		--credential_file=$env:GOOGLE_APPLICATION_CREDENTIALS
-#}
-#$env:DB_HOST=$env:TEST_CLOUDSQL2_HOST
-#$env:DB_USER=$env:TEST_CLOUDSQL2_SQLSERVER_USER
-#$env:DB_PASS=$env:TEST_CLOUDSQL2_SQLSERVER_PASS
-#$env:DB_NAME=$env:TEST_CLOUDSQL2_VOTES_NAME
-#try {
-	dotnet restore --force
-#	Receive-Job $proxy -ErrorAction 'SilentlyContinue'
-	dotnet build --no-restore
-#	Receive-Job $proxy -ErrorAction 'SilentlyContinue'
-#	try {
-#		Run-KestrelTest 5567 -CasperJs11
-#	} finally {
-#		Move-TestResults SqlServer
-#	}
-
-#} finally {
-#	Stop-Job $proxy
-#	Receive-Job $proxy -ErrorAction 'SilentlyContinue'
-#	Remove-Job $proxy
-#}
+if (-not (Test-Path cloud_sql_proxy.exe)) {
+    if ($PSVersionTable.Platform -eq 'Unix') {
+        Invoke-WebRequest -Uri 'https://dl.google.com/cloudsql/cloud_sql_proxy.linux.amd64' -OutFile cloud_sql_proxy.exe
+        chmod +x cloud_sql_proxy.exe
+    } else {
+        Invoke-WebRequest -Uri 'https://dl.google.com/cloudsql/cloud_sql_proxy_x64.exe' -OutFile cloud_sql_proxy.exe
+    }
+}
+$proxy = Start-Job -ArgumentList (Get-Location) -ScriptBlock {
+    Set-Location $args[0]
+    ./cloud_sql_proxy.exe --instances=$env:TEST_CLOUDSQL2_SQLSERVER_INSTANCE=tcp:1433 `
+        --credential_file=$env:GOOGLE_APPLICATION_CREDENTIALS
+}
+$env:INSTANCE_HOST=$env:TEST_CLOUDSQL2_HOST
+$env:DB_USER=$env:TEST_CLOUDSQL2_SQLSERVER_USER
+$env:DB_PASS=$env:TEST_CLOUDSQL2_VOTES_PASS
+$env:DB_NAME=$env:TEST_CLOUDSQL2_VOTES_NAME
+try {
+    dotnet restore --force
+    Receive-Job $proxy -ErrorAction 'Continue'
+    dotnet build --no-restore
+    Receive-Job $proxy -ErrorAction 'Continue'
+    try {
+        Run-KestrelTest 5567 -CasperJs11
+    } finally {
+        Move-TestResults SqlServer
+    }
+ 
+} finally {
+    Stop-Job $proxy
+    Receive-Job $proxy -ErrorAction 'Continue'
+    Remove-Job $proxy
+}

